oclint-rules/test/naming/LongVariableNameRuleTest.cpp

```diff
@@ -46,11 +46,11 @@ TEST_F(LongVariableNameRuleTest, ThreeCharsName)
 TEST_F(LongVariableNameRuleTest, FourCharsName)
 {
     testRuleOnCode(new LongVariableNameRule(), "void aMethod(int iiii) {}",
-        0, 1, 14, 1, 18, "Variable name with 4 characters is longer than the threshold of 3");
+        0, 1, 14, 1, 18, "Variable name with 4 characters (iiii) is longer than the threshold of 3");
```

Comment:
Now that you mention it, backticks (or other types of quotes) would definitely be better, I like that, yeah!  Applying changes very shortly.  ;)